### ISSUE TITLE
How to set lr for other dataset?

### ISSUE THREAD

**Issue by Huuush** [open-mmlab/mmdetection#4800]
Sry to bother. But i wanna to kwon, accroding to the docs in mmdet, "Important: The default learning rate in config files is for 8 GPUs and 2 img/gpu (batch size = 8*2 = 16)."  and the defalut lr is set to 0.01 for batch size = 16. So if i use the pretrain model, given by mmdet trained on coco, to trian other dataset, how can i set lr? if i also set my batch size to 16. Thx a lot!

**Comment by hhaAndroid** [comment]
Yes. Maybe you can refer to [finetune for cityscapes](https://github.com/open-mmlab/mmdetection/blob/master/configs/cityscapes/faster_rcnn_r50_fpn_1x_cityscapes.py)

**Comment by xiaosai666** [comment]
Hi, I also have a question about the epoch setting, I see the actual epoch=8*8 for the epoch comment. How did we get that? And where to find documentation for different types of Runners. Thank you very much!

**Comment by xiaosai666** [comment]
@hhaAndroid  Hope to get your help！

**Comment by hhaAndroid** [comment]
If you don't kown how to set LR, maybe you can refer to issue #4791.
